pootle/apps/pootle_app/project_tree.py

```diff
@@ -188,9 +196,12 @@ def create_or_resurrect_store(file, parent, name, translation_project):
         store_log(user='system', action=STORE_RESURRECTED,
                   path=store.pootle_path, store=store.id)
     except Store.DoesNotExist:
-        store = Store(file=file, parent=parent,
-                      name=name, translation_project=translation_project)
-
+        store = Store.objects.create(
+            file=file, parent=parent,
+            name=name, translation_project=translation_project)
+        store.filetype = ProjectFiletypes(
+            translation_project.project).choose_filetype(store.name)
```

Comment:
i think this logic is actually in the manager - so probs isnt necessary - ill check